frontend/css/artist-similarity.less

```diff
@@ -0,0 +1,251 @@
+
+    .artist-input-container {
+      height: inherit;
+      width: 45%;
+      display: flex;
+      flex-direction: column;
+
+      .searchbox-container {
+        display: flex;
+        flex-direction: row;
+        height: inherit;
+        width: 100%;
+
+        #searchbox-artist-name {
+          display: flex;
+          height: 100%;
+          width: 80%;
+          border-radius: 0;
+          border: 2px solid #e0e0e0;
+        }
+        #searchbox-icon {
+          vertical-align: middle;
+          background-color: @blue;
+          border: none;
+          height: inherit;
+          width: 20%;
+          min-width: fit-content;
+
+          &:hover {
+            background-color: @mb_blue_light;
+            border-color: @mb_blue_dark;
+          }
+        }
+      }
+      .searchbox-dropdown-container {
+        .searchbox-dropdown {
+          display: flex;
+          flex-direction: column;
+          max-height: 20vh;
+          overflow-y: scroll;
+          overflow-x: hidden;
+          box-shadow: 0px 6px 12px rgba(0, 0, 0, 0.175);
+          border: 1px solid #ccc;
+          border: 1px solid rgba(0, 0, 0, 0.15);
+          background-color: #e0e0e0;
+        }
+        .search-item {
+          display: block;
+          border: none;
+          text-align: left;
+          outline: inherit;
+          cursor: pointer;
+          background-color: @white;
+          z-index: 1;
+          padding: 4%;
+          font: inherit;
+          &:hover {
+            color: @white;
+            background-color: @blue;
+          }
+        }
+      }
+    }
+    .graph-size-input-container {
+      display: flex;
+      flex-direction: row;
+      height: 100%;
+      width: 45%;
+
+      #graph-size-input-label {
+        display: flex;
+        align-items: center;
+        padding-right: 0.5em;
+      }
+      #graph-size-input-number {
+        height: 100%;
+        width: 10%;
+        min-width: fit-content;
+        border-radius: 0;
+        border: 2px solid #e0e0e0;
+        border-left: none;
+        border-right: none;
+        text-align: center;
+      }
+      /*
+			* CSS to hide the up, down arrows for input number field
+			*/
+      input {
+        &::-webkit-outer-spin-button {
+          -webkit-appearance: none;
+          margin: 0;
+        }
+        &::-webkit-inner-spin-button {
+          -webkit-appearance: none;
+          margin: 0;
+        }
+      }
+      input[type="number"] {
+        -moz-appearance: textfield;
+      }
+
+      #graph-size-input-icon-minus,
+      #graph-size-input-icon-plus {
+        vertical-align: middle;
+        background-color: @blue;
+        border: none;
+        height: inherit;
+        width: 10%;
+        min-width: fit-content;
+        &:hover {
+          background-color: @mb_blue_light;
+          border-color: @mb_blue_dark;
+          cursor: pointer;
+        }
+      }
+    }
+  }
+  .artist-similarity-graph-panel-container {
```

Comment:
This doesn't look very nice on my screen; if there is not a lot of height everything is smooshed together and overlapping itself:
<img width="358" alt="image" src="https://github.com/metabrainz/listenbrainz-server/assets/6179856/bcd2dc15-c29b-4f69-823b-d3ef3031f971">

Please look into how to make this work better for other screen ratios (including mobile phones preferably)